manual/histograms/index.md

```diff
@@ -197,84 +268,114 @@ Creating a new histogram without changing the original one:
+
+{% highlight C++ %}
+   myHist->Draw("HIST")
 {% endhighlight %}
 
-#### Drawing options
 
-> **Note**
->
-> The drawing options are not case sensitive.
 
-**Drawing options for all histogram classes**
+## Drawing a histogram
+
+- Use the [TH1::Draw()](https://root.cern/doc/master/classTH1.html#aa53a024a9e94d5ec91e3ef49e49563da){:target="_blank"} method to draw a histogram using the provided drawing option.
+    The drawing is delegated to the {% include ref class="THistPainter" %} class that specializes the drawing of the histogram. The {% include ref class="THistPainter" %} class is separated
+    from the histogram, so that the histogram class does not contain the graphics overhead.
+
+### Drawing options
 
 The "drawing option" is the unique parameter of the [TH1::Draw()](https://root.cern/doc/master/classTH1.html#aa53a024a9e94d5ec91e3ef49e49563da){:target="_blank"}
 method. It specifies how the histogram will be graphically rendered.
 For detailed information on the drawing options for all histogram classes, refer to
 [THistPainter](https://root.cern/doc/master/classTHistPainter.html#HP01a){:target="_blank"}.
+For all possible available options see the [Histogram plotting options](https://root.cern/doc/master/classTHistPainter.html#HP01).
+
+> **Note**
+>
+> The drawing options are *NOT* case sensitive!
+
+### Drawing an histogram copy
+
+By default when an histogram is drawn in the ROOT canvas it is not copied in order to have automatically draw updates that can happen in the histogram object.
+This means that if the histogram object is created on the stack inside a defined C++ scope (or inside a Python function when using PyROOT), it will be automatically deleted when exiting the scope and
+the final consequence will be that the drawn object will disappear. To avoid this to happen you can use:
+
+  - [TH1::DrawCopy()](https://root.cern/doc/master/classTH1.html#aa19b24b96284284d677cd73f00d29d79){:target="_blank"} or [TObject::DrawClone()](https://root.cern/doc/master/classTObject.html#a7cd0f76ae1791c469f9472a9d4c8d6f9) methods to create a copy of the histogram when drawing it.
 
 _**Examples**_
 
 {% highlight C++ %}
-   TH2F h2("h2","Histogram filled with random numbers",40,-4,4,40,-20,20);
+   TH1D h1("h1","Histogram from a Gaussian",100,-3,3);
+   h1.FillRandom("gaus",10000);
+   h1.Draw();
+{% endhighlight %}
+
+{% include figure_jsroot
+   file="histograms.root" object="h1" width="500px" height="350px"
+   caption="Histogram drawn with Draw()."
+   %}
+
+- draw an histogram with error bars:
+
+{% highlight C++ %}
+   h1.Draw("E");
+{% endhighlight %}
+
+- draw a 2-D histogram as a LEGO plot:
+{% highlight C++ %}
+{
+   TH2D h2("h2","Histogram filled with random numbers",40,-4,4,40,-20,20);
    float px, py;
    for (int i = 0; i < 25000; i++) {
       gRandom->Rannor(px,py);
       h2.Fill(px,5*py);
    }
-   h2.Draw("LEGO1");
+   h2.DrawCopy("LEGO1");
```

Comment:
Why not using pointers ? that will avoid the use of `DrawCopy` ?

Reply:
I added this as an example when DrawCopy is needed